ecl/eclcc/eclcc.cpp

```diff
@@ -2462,7 +2462,10 @@ int EclCC::parseCommandLineOptions(int argc, const char* argv[])
         else if (iter.matchOption(tempArg, "-token"))
         {
             // For use by eclccserver - not documented in usage()
-            extractToken(tempArg, optWUID, StringAttrAdaptor(optUser), StringAttrAdaptor(optPassword));
+            StringBuffer wuid,user;
```

Comment:
Not sure how this is used?  Should we at least validate the signature?  Or is this purely a way of getting user and wuid and no need to validate because it's not relevant to security?